apps/ejabberd/src/mod_mongoose_admin.erl

```diff
@@ -184,7 +196,12 @@ registered_commands() ->
       } || C <- mongoose_commands:list(admin)].
 
 get_recent_messages(Caller, Other, Limit) ->
-    Res = lookup_recent_messages(Caller, Other, Limit),
+    {MegaSecs, Secs, _} = now(),
```

Comment:
`erlang:now/0` is deprecated. It's enough to take the time from `os:timestamp/0`